test/performance/scripts/create-outgoing-payments.js

```diff
@@ -1,5 +1,8 @@
 import http from 'k6/http'
 import { fail } from 'k6'
+import { createHMAC } from 'k6/crypto'
+import { canonicalize } from '/dist/json-canonicalize.bundle.js'
```

Comment:
```suggestion
import { canonicalize } from '../dist/json-canonicalize.bundle.js'
```
such that it also runs via local machine (not docker)

Reply:
committed locally